internal/server/worker.go

```diff
@@ -167,16 +166,16 @@ func (w *Worker) clone() *Worker {
 	cWorker := &Worker{
 		Worker: cw.(*store.Worker),
 	}
-	if w.apiTags != nil {
-		cWorker.apiTags = make([]*Tag, 0, len(w.apiTags))
-		for _, t := range w.apiTags {
-			cWorker.apiTags = append(cWorker.apiTags, &Tag{Key: t.Key, Value: t.Value})
+	if w.ApiTags != nil {
+		cWorker.ApiTags = make([]*Tag, 0, len(w.ApiTags))
+		for _, t := range w.ApiTags {
+			cWorker.ApiTags = append(cWorker.ApiTags, &Tag{Key: t.Key, Value: t.Value})
```

Comment:
It might be nice to create a `clone` method on `worker.Tags` which could then be used here in both places where we are cloning Tags.